ApplicationLibCode/ModelVisualization/RivGridPartMgr.cpp

```diff
@@ -196,7 +177,27 @@ void RivGridPartMgr::generatePartGeometry( cvf::StructGridGeometryGenerator& geo
 
     // Mesh geometry
     {
-        cvf::ref<cvf::DrawableGeo> geoMesh = geoBuilder.createMeshDrawable();
+        cvf::ref<cvf::DrawableGeo> geoMesh;
+
+        if ( RiaPreferencesGrid::current()->radialGridMode() == RiaGridDefines::RadialGridMode::USE_CYLINDRICAL && m_grid->isRadial() &&
+             m_grid->isTempGrid() )
+        {
+            std::vector<size_t> localGridCellIndices;
+            for ( size_t i = 0; i < m_grid->cellCount(); i++ )
+            {
+                if ( !( *m_cellVisibility )[i] ) continue;
+                localGridCellIndices.push_back( i );
```

Comment:
Uncesseary use of `continue` keyword here?

Reply:
Fixed